sqlglot/parser.py

```diff
@@ -6517,6 +6512,30 @@ def _parse_alter(self) -> exp.AlterTable | exp.Command:
 
         return self._parse_as_command(start)
 
+    def _parse_alter_view(self, start) -> exp.AlterView | exp.Command:
+        self._parse_select()
+        table_parts = self._parse_table_parts()
+        this = self._parse_schema(this=table_parts)
```

Comment:
I think we can consolidate these two, imitating the current implementation for `ALTER TABLE`:

```suggestion
        this = self._parse_table(schema=True)
```

Reply:
Updated